stable/jupyter/templates/jupyter-deployment.yaml

```
@@ -177,3 +180,6 @@ spec:
 {{- if .Values.volumes }}
 {{ include .Values.volumes.volumesTemplate . | indent 8 }}
 {{- end }}
+{{- if .Values.pvc }}
+{{ include .Values.pvc.pvcTemplate . | indent 8 }}
```

```suggestion
{{ include .Values.pvc.pvcVolumesTemplate . | indent 8 }}
```
I suppose? to differentiate and keep the naming conventions established

---

ok